package_modules/apt_get

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/python
+
+import sys
+import os
+import subprocess
+import re
+
+dpkg_cmd = "/usr/bin/dpkg"
+dpkg_deb_cmd = "/usr/bin/dpkg-deb"
+dpkg_query_cmd = "/usr/bin/dpkg-query"
+dpkg_output_format = "Name=${Package}\nVersion=${Version}\nArchitecture=${Architecture}\n"
+dpkg_status_format = "Status=${Status}\n" + dpkg_output_format
+
+apt_get_options = ["-o", "Dpkg::Options::=--force-confold", "-o", "Dpkg::Options::=--force-confdef", "-y", "--force-yes"]
```

Comment:
This would read more clearly with some line-breaks, so that each option and optional parameter appear on one line together, separate from the others.
```
apt_get_options = ["-o", "Dpkg::Options::=--force-confold",
                 "-o", "Dpkg::Options::=-force-confdef",
                 "-y",
                 "--force-yes"]
```
which would even make it easy for you add comments explaining what each is for and why we need it.